content/06-vets-developer-docs/04-vets-website/03-forms/02-common-definitions.md

```diff
@@ -56,7 +56,7 @@ Here we're using Javascript's object shorthand syntax to set the `email` propert
 
 ## Using a common front-end definition
 
-We have a collection of common definitions and fields on the front-end for you to use in your form, which are detailed in our [field library doc](https://github.com/department-of-veterans-affairs/vets-website/blob/master/docs/schemaform/field-library.md). Let's try using our common full name field:
+We have a collection of [common definitions](https://github.com/usds/us-forms-system/blob/master/docs/building-a-form/common-definitions.md) and [fields](https://github.com/usds/us-forms-system/blob/master/docs/building-a-form/available-widgets.md) on the front-end for you to use in your form. Let's try using our common full name field:
```

Comment:
My concern at this point is fixing what's currently a broken link. These two links seem to be the latest version of what the link used to point to. But I'm not 100% sure since I can't find a file named `field-library.md` anywhere.

Reply:
We could remove this for now, since things are in flux, and create a todo item to see how the documentation can supplement the example templates for now.

Reply:
okay I'll remove the links entirely for now.